src/dndbeyond/base/utils.js

```diff
@@ -485,6 +513,14 @@ async function sendRoll(character, rollType, fallback, args) {
         }
     }
 
+    // effects 
+    if((rollType == "saving-throw" || rollType == "ability" || rollType == "skill") && req.ability == "STR" && character.getSetting("effects-enlarge", false)) {
+        adjustRollAndKeyModifiersWithAdvantage(req);
+        addEffect(req, "Enlarge");
+    } else if((rollType == "saving-throw" || rollType == "ability") && req.ability == "STR" && character.getSetting("effects-reduce", false)) {
```

Comment:
Missing the check for "skill" on the reduce, you only added it on enlarge.
Also, could benefit from using `["saving-throw", "ability", "skill"].includes(rollType)` for cleaner code readability